CotEditor/Configurations/CotEditor-Sparkle.xcconfig

```diff
@@ -0,0 +1,31 @@
+//
+//  CotEditor-AppStore.xcconfig
+//
+//  CotEditor
+//  https://coteditor.com
+//
+//  Created by Yoshimasa Niwa on 4/7/20.
+//
+//  ---------------------------------------------------------------------------
+//
+//  © 2020 CotEditor Project
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  https://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+#include "CotEditor.xcconfig"
```

Comment:
It seems there is no need to separate CotEditor-AppStore.xcconfig from CotEditor.xcconfig at the moment. I suppose it's better to remove and add again when it becomes needed.